src/vs/workbench/contrib/terminalContrib/chatAgentTools/common/terminalChatAgentToolsConfiguration.ts

```diff
@@ -20,6 +20,8 @@ export const enum TerminalChatAgentToolsSettingId {
 	DeprecatedAutoApprove2 = 'chat.agent.terminal.denyList',
 	DeprecatedAutoApprove3 = 'github.copilot.chat.agent.terminal.allowList',
 	DeprecatedAutoApprove4 = 'github.copilot.chat.agent.terminal.denyList',
+
+	AutoReplyToPrompts = 'chat.tools.terminal.experimental.autoReplyToPrompts',
```

Comment:
Let's stick this in the list above, you can remove the space before ShellIntegrationTimeout. I just wanted the `Deprecated*` ones separated